x-pack/plugins/translations/translations/fr-FR.json

```diff
@@ -6783,74 +6783,6 @@
     "searchConnectors.syncStatus.inProgress": "Synchronisation en cours",
     "searchConnectors.syncStatus.pending": "Synchronisation en attente",
     "searchConnectors.syncStatus.suspended": "Synchronisation suspendue",
-    "searchConnectorsPlugin.content.nativeConnectors.azureBlob.description": "Effectuez des recherches sur votre contenu sur Stockage Blob Azure.",
```

Comment:
Can we rename these translations to the new path rather than deleting them?

Reply:
Sorry I assumed that whatever `node scripts/i18n_check.js --fix` does is correct. I'll rename the keys.